cluster_experiments/experiment_analysis.py

```diff
@@ -680,3 +682,226 @@ def analysis_point_estimate(self, df: pd.DataFrame, verbose: bool = False) -> fl
         """
         results_mlm = self.fit_mlm(df)
         return results_mlm.params[self.treatment_col]
+
+
+class SyntheticControlAnalysis(ExperimentAnalysis):
+    """
+    Class to run Synthetic control analysis. It expects only one treatment cluster.
+
+    Arguments:
+
+        target_col (str): The name of the column containing the variable to measure.
+        treatment_col (str): The name of the column containing the treatment variable.
+        treatment (str): The name of the treatment to use as the treated group.
+        cluster_cols (list): A list of columns to use as clusters.
+        hypothesis (str): One of "two-sided", "less", "greater" indicating the hypothesis.
+        time_col (str): The name of the column containing the time data.
+        intervention_date (str): The date when the intervention occurred.
+    Usage:
+
+    ```python
+    from cluster_experiments.experiment_analysis import SyntheticControlAnalysis
+    import pandas as pd
+    import numpy as np
+
+    dates = pd.date_range(start_date, end_date, freq="d")
+
+    users = [f"User {i}" for i in range(N)]
+
+    # Create a combination of each date with each user
+    combinations = list(product(users, dates))
```

Comment:
do you need to import product in here?

Reply:
yep, this is not failing because you need to add the class in test_docs.py. You also need to add it init in the cluster_experiments folder for it to be easier to import, and in readme for it to be more visible.